src/code/z_message.c

```diff
@@ -526,7 +527,7 @@ void func_80152EC0(PlayState* play) {
     if (1) {}
     if ((msgCtx->songPlayed < 0x17) && (msgCtx->songPlayed != 0xE) &&
         ((msgCtx->ocarinaAction < 0x43) || (msgCtx->ocarinaAction >= 0x47))) {
-        msgCtx->unk120B0 = 1;
+        msgCtx->ocarinaSongEffectActive = 1;
```

Comment:
```suggestion
        msgCtx->ocarinaSongEffectActive = true;
```
Bool as well for this